src/components/text-input/index.md.njk

```diff
@@ -27,7 +27,7 @@ There are 2 ways to use the text input component. You can use HTML or, if you’
 
 ### Label text inputs
 
-All text inputs must have visible labels; placeholder text is not an acceptable replacement for a label as it vanishes when users click on the text input.
+All text inputs must have visible labels; placeholder text is not an acceptable replacement for a label as it vanishes when users type in the text input.
```

Comment:
Might be able to simplify it a little:

```suggestion
All text inputs must have visible labels; placeholder text is not an acceptable replacement for a label as it vanishes when users start typing.
```

Thoughts?

Reply:
yeh looks good - do I press that commit suggestion button?

Reply:
If you like!